src/pe_reports/data/data_schema.sql

```diff
@@ -558,6 +558,35 @@ CREATE TABLE public.unique_software (
     software_name text NOT NULL
 );
 
+--
+--Name: was_tracker_customerdata
+--
+
+create table was_tracker_customerdata
+(
+    customer_id              uuid default uuid_generate_v1() not null
+        constraint was_tracker_customerdata_pk
+            primary key,
+    tag                      text                            not null,
+    customer_name            text                            not null,
+    testing_sector           text                            not null,
+    ci_type                  text                            not null,
+    ticket                   text                            not null,
+    next_scheduled           text                            not null,
+    last_scanned             text                            not null,
+    frequency                text                            not null,
+    comments_notes           text                            not null,
+    was_report_poc           text                            not null,
+    was_report_email         text                            not null,
+    onboarding_date          text                            not null,
+    no_of_web_apps           integer                         not null,
+    no_web_apps_last_updated text,
+    elections                text
```

Comment:
Looks like you are using different spacing and capitalization (e.g. `not null` vs. `NOT NULL`) than the rest of this file.  These things should be consistent throughout the file.

Reply:
@dav3r , 
I have updated to capitalization per commit [c8befc4](https://github.com/cisagov/pe-reports/pull/427/commits/c8befc498db0e2d859499ea0a636994f534b8e3d)

Reply:
What about the spacing differences here vs. the rest of the file?